frontend/apps/reader/skimtowidget.lua

```diff
@@ -176,7 +176,7 @@ function SkimToWidget:init()
         show_parent = self,
         callback = function()
             self.curr_page = self.curr_page + 10
-           self:update()
+	    self:update()
```

Comment:
Ah no, it should be eight spaces, not a tab followed by four spaces. :-)

Reply:
sorry for that..